Gemfile

```diff
@@ -2,30 +2,27 @@
 
 source "https://rubygems.org"
 
-DECIDIM_VERSION = "release/0.24-stable"
+DECIDIM_VERSION = "release/0.26-stable"
 
 ruby RUBY_VERSION
 
 gem "decidim", git: "https://github.com/decidim/decidim.git", branch: DECIDIM_VERSION
 
-gem "decidim-decidim_awesome", "~> 0.7.0"
-gem "decidim-term_customizer", git: "https://github.com/mainio/decidim-module-term_customizer.git", branch: "master"
-gem "omniauth-publik", git: "https://github.com/OpenSourcePolitics/omniauth-publik", branch: "v0.0.9"
+# gem "decidim-decidim_awesome", "~> 0.7.0"
+# gem "decidim-term_customizer", git: "https://github.com/mainio/decidim-module-term_customizer.git", branch: "master"
+# gem "omniauth-publik", git: "https://github.com/OpenSourcePolitics/omniauth-publik", branch: "v0.0.9"
```

Comment:
Could you uncomment those 3 gems and update the required branch? We still need them for the 0.26 version of this platform.